frontend/src/css/pages/home.scss

```diff
@@ -0,0 +1,58 @@
+@import '../settings/global';
+@import '../components/_button';
+
+.intro{
+    height: 616px;
```

Comment:
Never give fixed height and width. Height will increase or decrease automatically depending on the content. Width will increase/decrease depending on the screen size.

Reply:
Okay.. shall I use padding instead?